src/river.py

```
@@ -0,0 +1,292 @@
+    QgsGeometry,
+    QgsField,
+    QgsRaster,
+    QgsFeature,
+    QgsApplication
+)
+from processing_saga_nextgen.saga_nextgen_plugin import SagaNextGenAlgorithmProvider
+from qgis.PyQt.QtCore import QVariant
+import processing
+import requests
+from qgis.utils import iface
+from .common import *
```

Ruff (0.8.2)

`from .common import *` used; unable to detect undefined names

(F403)